docs/toc.yml

```
@@ -674,51 +674,50 @@
     - name: Overview
       href: excel/custom-functions-overview.md
       displayName: Excel, Custom Functions
-    - name: Get started with Excel custom functions
-      href: quickstarts/excel-custom-functions-quickstart-redirect.md
-      displayName: Excel, Custom Functions
-    - name: Custom functions tutorial
-      href: tutorials/excel-tutorial-create-custom-functions.md
+    - name: CustomFunctionsRuntime API requirement sets
+      href: excel/custom-functions-requirement-sets.md
       displayName: Excel, Custom Functions
-    - name: Shared custom function data with the task pane tutorial
-      href: tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md
+    - name: Tutorials
+      items:
+      - name: Build your first custom functions add-in
+        href: quickstarts/excel-custom-functions-quickstart-redirect.md
+        displayName: Excel, Custom Functions
+      - name: Custom functions tutorial
+        href: tutorials/excel-tutorial-create-custom-functions.md
+        displayName: Excel, Custom Functions
+      - name: Share custom function data with the task pane tutorial
+        href: tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md
+        displayName: Excel, Custom Functions
+    - name: Naming guidelines
+      href: excel/custom-functions-naming.md
       displayName: Excel, Custom Functions
-    - name: Call Excel JS APIs from your custom function
-      href: excel/call-excel-apis-from-custom-function.md
+    - name: Parameter options
+      href: excel/custom-functions-parameter-options.md
       displayName: Excel, Custom Functions
     - name: Error handling
       href: excel/custom-functions-errors.md
       displayName: Excel, Custom Functions
+    - name: Autogenerate JSON metadata for custom functions
```

```suggestion
    - name: Autogenerate JSON metadata
```
When were under the "custom functions" node I think we can leave that part out.